### PR TITLE
Feature - move to hosted swagger instance

### PR DESCRIPTION
**Pull request by krishan-carbon**
this will let us have like `https://swagger.arkera.co?url=https://penguin.arkera.co/swagger-file` so instead of having swagger copied into each repo, they just redirect to here. The benefit is python-core becomes cleaner and each service doesn't have to worry about swagger hosting (api times etc). also, we can use this to preview prs much easier by having url point to the raw pr swagger file. The work required to achieve this is that each services swagger file will need to allow cross-origin requests.

### CODE REVIEW

**Comment by krishan-carbon** [comment]
oh also, I have bundled the editor in so we have a safe (trusted) place to edit our swagger file.

**Comment by krishan-carbon** [comment]
(and we can then make this repo private again)

**Comment by krishan-carbon** [comment]
** PLEASE DONT MERGE ** its based against a branch where i have deleted everything so you can see just the important changes

**Comment by krishan-carbon** [comment]
Notes on the new build file:
- no upper level and lower level one differences
- just one file will be used everywhere (so will add the pypi deployment, unittest running, integration test running etc here)
- vars files are separate and provide the context for the build file and are `source`d in

File: src/production.vars [hunk 1]
```diff
@@ -0,0 +1,17 @@
+AWS_ID=367793910174
+DOCKER_REPO=${AWS_ID}.dkr.ecr.eu-west-1.amazonaws.com
+
+SERVICE_NAME=swagger
+ENVIRONMENT=production
+ECS_CLUSTER=carbon-worker-cluster
+ECS_NAME=${SERVICE_NAME}
+
+VERSION=`git rev-list --count HEAD`
+MEMORY_LIMIT_MB=300
```
Comment (garyjames7) [inline src/production.vars]: :+1:

**Comment by garyjames7** [comment]
@krishan-carbon - btw, think you're the only person who can merge this anyway

**Comment by krishan-carbon** [comment]
everyone should be able to now

**Comment by krishan-carbon** [comment]
@garyjames7

**Comment by krishan-carbon** [comment]
oh wait python-core needs an update before this goes in.

**Pull request merged**